src/logic/operate.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable-next-line */
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  switch (operation) {
+    case '÷':
+      return Big(numberOne).div(numberTwo).toString();
```

Comment:
If `NumberTwo` is zero`(0)` then this line would throw a `DividedByZero` Error.
Kindly handle this error

Reply:
Thanks! I will fix the errors